integration-tests/src/test/java/oracle/kubernetes/operator/ItOperatorUpgrade.java

```diff
@@ -49,11 +54,24 @@
    */
   @BeforeAll
   public static void staticPrepare() throws Exception {
+    namespaceList = new StringBuffer();
     testClassName = new Object() {
     }.getClass().getEnclosingClass().getSimpleName();
     initialize(APP_PROPS_FILE, testClassName);
   }
 
+  /**
+   * Releases k8s cluster lease, archives result, pv directories.
+   *
+   * @throws Exception exception
```

Comment:
yeah, i know it is an exception, you are meant to explain why

Reply:
This is standard junit lifecycle method being used across all test classes.   What you want me to explain more here.

Reply:
which is better:
``` @throws Exception exception```
or 
``` @throws Exception if unable to collect diagnostics and clean up resources.```

Just because it is done wrong elsewhere, does not make it right to do it wrong here.

Reply:
or, you could just say `@throws Exception @see BaseTest#tearDown`

Reply:
Added 
@throws Exception if tearDown() fails
@see BaseTest#tearDown